src/App.vue

```diff
@@ -670,6 +667,9 @@ export default class App extends Mixins(
         throw error // go to catch()
       }
 
+      // set completing party
+      this.setCompletingParty(this.getCompletingParties())
```

Comment:
Auth info (ie, whether user is staff) was fetched in `handleDraftDissolution()` or `handleDraftApplication()`, called above.